crates/runtime/src/memory.rs

```diff
@@ -221,6 +226,9 @@ pub enum Memory {
     Dynamic(Box<dyn RuntimeLinearMemory>),
 }
 
+unsafe impl Send for Memory {}
+unsafe impl Sync for Memory {}
```

Comment:
If this is still necessary then I think this should be done at a lower level where it's more obvious which field causes the automatic inference to not work and a comment can say why this is needed.

Reply:
Ah, it no longer is, now that we're taking the more idiomatic pass-ownership-of-`MemFdSlot` approach. Removed!